.pre-commit-config.yaml

```diff
@@ -80,7 +80,7 @@ repos:
         additional_dependencies: [toml]
 
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 22.10.0
```

Comment:
Seems like a big jump. Will this affect our APE 20 campaign?

Reply:
Oh, 22.10 was released in October, and 22.3 in March ... would have been better to get this update before the big conversion.

Reply:
It shouldn't effect anything. I've been doing my black changes with the latest version directly. 

I can rebase all the PRs before we merge as this and the current changes on main shouldn't cause any merge conflicts.

Reply:
@saimn , shall we merge?

Reply:
Yep, OK. I'm also using 22.10 locally. It seems there were some changes in the `--preview` mode but should be fine.